Makefile

```diff
@@ -885,7 +885,11 @@ cpplint:
 	@$(PYTHON) tools/check-imports.py
 
 ifneq ("","$(wildcard tools/eslint/lib/eslint.js)")
-lint: jslint cpplint
+lint:
+	EXIT_STATUS=0 ; \
+	$(MAKE) jslint || EXIT_STATUS=$$? ; \
+	$(MAKE) cpplint || EXIT_STATUS=$$? ; \
+	exit $$EXIT_STATUS
 CONFLICT_RE=^>>>>>>> [0-9A-Fa-f]+|^<<<<<<< [A-Za-z]+
 lint-ci: jslint-ci cpplint
```

Comment:
@aqrln I've updated `lint` with idea to get all the lint error when I run `lint` or `test` locally.
On CI it is enough to know the fact that something (and what exactly) failed. That's why I didn't touch `lint-ci`. Do you think it should be updated also?